streams/iq.m3u

```diff
@@ -3,16 +3,16 @@
 http://rtmp1.abnsat.com/hls/arabic.m3u8
 #EXTINF:-1 tvg-id="AfarinTV.iq",Afarin TV (720p) [Not 24/7]
 https://5ab29cc78f681.streamlock.net/afarinTV/livestream/playlist.m3u8
-#EXTINF:-1 tvg-id="AlEtejah.iq",Al-Etejah (576p) [Not 24/7]
-http://streaming.aletejahtv.iq:1935/etejah-newLive/smil:etejahNew.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="AlghadeerTV.iq",AlGhadeer (720p) [Not 24/7]
-https://sd343444.vodu.store:3356/live/Alghadeer/index.m3u8
+#EXTINF:-1 tvg-id="AlforatTV.iq",Alforat TV (1080p) 
+http://95.216.180.111:1935/live/10/playlist.m3u8
+#EXTINF:-1 tvg-id="AlhurraIraq.us",Alhurra Iraq (720p)
```

Comment:
Should be in the US.m3u playlist

Reply:
No, it is an Iraqi channel, speaking in Arabic, but it broadcasts from inside America, so it is not an American channel.

Reply:
It's based in the US
https://en.wikipedia.org/wiki/Alhurra